pyproject.toml

```diff
@@ -45,7 +45,6 @@ pylint-flask = "0.6"
 requirements-detector = ">=1.3.1"
 PyYAML = "*"
 mccabe = "^0.7.0"
-flake8 = "*"
```

Comment:
> So in fact we should have removed it when it dropped python 3.7 instead of limiting the version.
Probably yes :-)

> ruff could have replaced flake8 / pyflakes / pycodestyle.
Yes, but it's another subject, we can open an issue about that :-)